utils/update_build_version.py

```diff
@@ -19,14 +19,18 @@
 #
 # Args: <spirv-tools_dir>
 #
-# For each directory, there will be a line in build-version.inc containing that
-# directory's "git describe" output enclosed in double quotes and appropriately
-# escaped.
+# For each directory, there will be a line in build-version.inc containing two
```

Comment:
Unclear what "each directory" refers to if there's a single argument.  (Sorry, this was my fault in the original version.)